thruster_control/include/thruster_control/thruster_control.h

```diff
@@ -34,39 +34,35 @@
 
 // Original version: Christopher Scianna Christopher.Scianna@us.QinetiQ.com
 
-
 /*
  * thruster_control.h
  */
 
-#ifndef _THRUSTER_CONTROL_H_
-#define _THRUSTER_CONTROL_H_
-
-#include <std_msgs/String.h>
-#include <std_msgs/UInt16.h>
-#include <std_msgs/UInt8.h>
-#include "std_msgs/Header.h"
+#ifndef THRUSTER_CONTROL_THRUSTER_CONTROL_H
+#define THRUSTER_CONTROL_THRUSTER_CONTROL_H
 
 #include <ros/ros.h>
 #include <cmath>
 #include <diagnostic_tools/diagnosed_publisher.h>
 #include <diagnostic_tools/health_check.h>
+#include <diagnostic_tools/message_stagnation_check.h>
+#include <diagnostic_tools/periodic_message_status.h>
 #include <diagnostic_updater/diagnostic_updater.h>
-
+#include <health_monitor/ReportFault.h>
 #include <thruster_control/ReportMotorTemperature.h>
 #include <thruster_control/ReportRPM.h>
 #include <thruster_control/SetRPM.h>
-
-#include <health_monitor/ReportFault.h>
-
 #include "CANIntf.h"
```

Comment:
@patapufate shouldn't this be 
```suggestion
#include "thurster_control/CANIntf.h"
```
? Also, should we be installing this? I wonder if we shouldn't put these implementation details under `src` (and keep `src` as a private build time include directory).

Reply:
Fixed. Added "Thruster_Control"